terraform/digitalocean/k8s/backend.tf

```diff
@@ -0,0 +1,9 @@
+# INFO Read about spaces here https://www.digitalocean.com/docs/spaces/
+terraform {
+  backend "s3" {
+    skip_requesting_account_id  = true
+    skip_credentials_validation = true
+    skip_get_ec2_platforms      = true
+    skip_metadata_api_check     = true
+  }
+}
```

Comment:
This should be in `terraform.tf` file

You can use [this](https://github.com/shalb/cluster.dev/blob/cb2a13efd4caa944c67d7df5b912ce025525f0ce/terraform/aws/minikube/terraform.tf) as example